client/components/hello-world.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
-export default function HelloWorld(props) {
+export default function openingButton(props) {
```

Comment:
Make sure your components names start with a capital letter.

Reply:
Whoops! Will update that right now, thanks!